specs/sifive.com/SCIE/pipelined/0.1.0/SCIEPipelined.json5

```
@@ -0,0 +1,59 @@
+{
+  abstractionDefinition: {
+    vendor: 'sifive.com',
+    library: 'core',
+    name: 'SCIEPipelined_rtl',
+    version: '0.1.0',
+    busType: {
+      vendor: 'sifive.com',
+      library: 'core',
+      name: 'SCIEPipelined',
+      version: '0.1.0',
+    },
+    ports: {
+      clock: {
+        description: 'must be used to clock a positive-edge-triggered pipeline register. The functional unit must contain exactly one stage of pipeline register. for two-cycle SCIE only.',
```

```suggestion
        description: 'Must be used to clock a positive-edge-triggered pipeline register. The functional unit must contain exactly one stage of pipeline register. For two-cycle SCIE only.',
```

If we're going to properly capitalizes sentences for part of the description, then we should properly capitalize the whole description.

---

fixed